app/services/action_handlers/repository.rb

```diff
@@ -0,0 +1,32 @@
+module ActionHandlers
```

Comment:
Any reason why this file isn't tested?

Reply:
I added tests for this file in the other PR https://github.com/rootstrap/rs-code-review-metrics/pull/442